iOS Client/Info.plist

```diff
@@ -19,7 +19,7 @@
 	<key>CFBundleShortVersionString</key>
 	<string>1.0</string>
 	<key>CFBundleVersion</key>
-	<string>1</string>
+	<string>5</string>
```

Comment:
why 5?

Reply:
because I tried to build 4 times before this one lol Every time you upload to app store it has to be a unique build number and I messed it up 4 times